offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/person/create/CreatePersonViewModel.groovy

```diff
@@ -51,8 +51,7 @@ class CreatePersonViewModel {
         availableAffiliations = new ObservableList(new ArrayList<Affiliation>(affiliationService.iterator().collect()))
 
         this.affiliationService.subscribe({
-            this.availableAffiliations.clear()
-            this.availableAffiliations.addAll(affiliationService.iterator().collect())
+            this.availableAffiliations.add(it)
```

Comment:
This might not be correct. The affiliation service also fires for removal

Reply:
```suggestion
            if(it) this.availableAffiliations.add(it)
```

Reply:
I think we added the availableaffiliation.clear() call since otherwise the selected affiliation appears in the create customer view.

Reply:
@Steffengreiner I think that was not the issue. The create customer affiliation selection was due to a listener that is now removed. The clear was added to avoid duplicate entries AFAIK

Reply:
If the `equals` method is implemented correctly we could also do
```suggestion
            if (! (it in this.availableAffiliations) ) this.availableAffiliations.add(it)
```

Reply:
Good Idea 👍